composer.json

```diff
@@ -100,6 +101,7 @@
         "sensio/framework-extra-bundle": "^5.0.0",
         "shudrum/array-finder": "^1.1.0",
         "smarty/smarty": "^3.1.31",
+        "soundasleep/html2text": "^0.5.0",
```

Comment:
probably because of PHP 5.6 support, right? :cry:

Reply:
what do you mean? This is used to convert html templates into text templates

Reply:
oh you mean the version number which is not 1.1.0?
I didn't even notice I let composer define this, but yes it probably comes from the PHP requirements.